tests/test_layout.py

```diff
@@ -1212,6 +1212,14 @@ def test_focused_frame_child(hlwm):
         assert hlwm.get_attr('tags.0.tiling.focused_frame.index') == focused_index
 
 
+def verify_frame_indices(hlwm, root='tags.focus.tiling.root', index=[]):
```

Comment:
Could you add a short docstring on what and how this function does what it does?
I found it not that obvious (but that could be because I didn't have enough coffee just yet)

Reply:
I've added a docstring. :-)

Reply:
Thank you, it is so much clearer now :)